pkg/static/login.js

```diff
@@ -422,7 +424,7 @@
             show("#login-wait-validating");
             const xhr = new XMLHttpRequest();
             xhr.open("GET", login_path, true);
-            xhr.setRequestHeader("Authorization", "Bearer " + token_val);
+            xhr.setRequestHeader(authorization_header, "Bearer " + token_val);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17473-20220811-104722-2893e7f4-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)